examples/two-step-transfer/src/lib.rs

```diff
@@ -632,4 +633,62 @@ mod tests {
         claim!(host.state().requests.get(&request_id_outdated).is_some());
         claim!(host.state().requests.get(&request_id_active).is_some());
     }
+
+    #[concordium_quickcheck]
+    fn prop_receive_support_transfer(
```

Comment:
Documentation please.

Reply:
I've added a property statement, it's a bit long, and it probably could be more general. I'll work on better examples in a separate PR.